owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -445,13 +448,15 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
         val customTabsBuilder: CustomTabsIntent.Builder = CustomTabsIntent.Builder()
         val customTabsIntent: CustomTabsIntent = customTabsBuilder.build()
 
+        this.oidcState = UUID.randomUUID().toString().substring(0,15)
```

Comment:
Isn't it better to use base64 encoded bytes from SecureRandom here?

Reply:
definitely better, created a new function in OauthUtils for that